developer/upgrading/0.24.0.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Upgrading to 0.24.0
+---
+
+If you haven't upgraded to `0.23.0` please read the [upgrading
+guide](/developer/upgrading/0.23.0).
+
+`v0.24.0` of the runtime rethinks the way we handle draft/preview mode. Refer to
+the [official release
+notes](https://github.com/makeswift/makeswift/releases/tag/%40makeswift/runtime%400.24.0)
```

Comment:
This link doesn't appear to work yet. Is that correct? Is that going to be published soon?